src/panels/config/integrations/integration-panels/zwave_js/zwave_js-config-dashboard.ts

```diff
@@ -83,7 +81,17 @@ class ZWaveJSConfigDashboard extends SubscribeMixin(LitElement) {
 
   protected firstUpdated() {
     if (this.hass) {
-      this._fetchData();
+      this._fetchData().then(() => {
+        if (this._status === "connected") {
+          const inclusion_state = this._network?.controller.inclusion_state;
+          // show dialog if inclusion/exclusion is already in progress
+          if (inclusion_state === InclusionState.Including) {
+            this._addNodeClicked();
+          } else if (inclusion_state === InclusionState.Excluding) {
+            this._removeNodeClicked();
+          }
+        }
+      });
```

Comment:
```suggestion
      await this._fetchData()
      if (this._status === "connected") {
        const inclusion_state = this._network?.controller.inclusion_state;
        // show dialog if inclusion/exclusion is already in progress
        if (inclusion_state === InclusionState.Including) {
          this._addNodeClicked();
        } else if (inclusion_state === InclusionState.Excluding) {
          this._removeNodeClicked();
        }
      }
```

and edit `protected async firstUpdated() {`